packages/editor/src/components/post-publish-panel/maybe-category-panel.js

```diff
@@ -23,12 +23,15 @@ function MaybeCategoryPanel() {
 		const postType = select( editorStore ).getCurrentPostType();
 		const categoriesTaxonomy =
 			select( coreStore ).getTaxonomy( 'category' );
-		const defaultCategorySlug = 'uncategorized';
+		const defaultCategoryId = select( coreStore ).getEntityRecords(
```

Comment:
```suggestion
		const defaultCategoryId = select( coreStore ).getEntityRecord(
```

The `getEntityRecords` records will return an array of the same options.